locales/en-US/snoozetabs.properties

```diff
@@ -1,35 +1,45 @@
-# LOCALIZATION NOTE: This add-on will let you select tabs to close for a while, after which they will re-appear in your browser with a notification.
+# This add-on will let you select tabs to close for a while, after which they
+# will re-appear in your browser with a notification.
 extDesc=An add-on to let you snooze your tabs for a while.
 popupTitle=Snooze tab until…
 contextMenuTitle=Snooze tab until…
 bookmarkFolderTitle=Snoozed Tabs
 notificationTitle=Tab woke up…
 
+# Alternative text for the tab icon
 confirmIconAltText=tab icon
-# LOCALIZATION NOTE: "{time}" is the time the tab will snooze until, as per https://github.com/bwinton/SnoozeTabs/issues/60
-confirmTimeTitle=This tab will snooze until {time}.
+
+# "{time_exp}" is the time the tab will snooze until, see also
```

Comment:
`{time_exp}` doesn't work out of the box. I haven't debugged why, but I assume because of [this regex](https://l10n.mozilla-community.org/webstatus/?product=activity-stream). Can look into it if needed.